### DISCUSSION TITLE
Webrtc to rtsp stream

### DISCUSSION THREAD

**Discussion by PalmDomenico** [bluenviron/mediamtx#3826] [Questions]
### Question

Hi, i use a client web for getting the webcam source and transmitting this to server with webrtc, can i use this repo for converting webrtc source to rtsp source?

**Comment by buyiasd** [comment]
I would also like to know the answer to this question. Thanks

**Comment by aler9** [comment]
As written inside the README, conversion between protocols is automatic.
The README contains instructions on how to publish streams with WebRTC.
The README also contains instructions on how to read streams with RTSP.
Combine the two things and you'll get what you want.
